src/Network/Network/Models/PSVpnClientConnectionHealthDetail.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.WindowsAzure.Commands.Common.Attributes;
+
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public partial class PSVpnClientConnectionHealthDetail
+    {
+        [Ps1Xml(Label = "VpnConnectionId", Target = ViewControl.Table)]
```

Comment:
For these attributes to be useful, you need to generate the format.ps1xml from the models. See documentation here on how to do that: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/generate-format.ps1xml-file.md#how-to-generate-formatps1xml-file